packages/language-support/src/tests/autocompletion/databasenameCompletion.test.ts

```diff
@@ -15,6 +27,32 @@ describe('Can complete database names', () => {
     },
   };
 
+  test('Correctly completes database names/aliases with special symbols using backticks', () => {
+    const query = 'SHOW DATABASE ';
```

Comment:
This test is unnecessary, you could just add the completions to the existing test?

Reply:
I split it on purpose so you would see that only this test failed if something purely related to backticks broke. You don't think there's any value to this?